src/internal/components/InternalEditor.tsx

```diff
@@ -97,41 +96,62 @@ export const InternalEditor = ({
   };
 
   const handleSave = async (data: Data) => {
+    if (themeModeActive) {
+      devLogger.logFunc("saveStyles");
+      // save styles
```

Comment:
is this commenting indicating the styles are being saved?

Reply:
Meant it as a stub for where we'll hook up things later -- updated to make that clearer!